BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.m

```diff
@@ -189,6 +187,32 @@ - (BOXURLSessionTaskCachedInfo *)cachedInfoForUserId:(NSString *)userId associat
     return cachedInfo;
 }
 
+- (NSString *)destinationFilePathGivenBackgroundSessionId:(NSString *)backgroundSessionId sessionTaskId:(NSUInteger)sessionTaskId
+{
+    NSString *filePath = [self filePathForBackgroundSessionId:backgroundSessionId sessionTaskId:sessionTaskId type:BOXURLSessionTaskCacheFileTypeDestinationFilePath];
+
+    //decrypt data found at filePath
+    NSData *data = [self unencryptedDataAtFilePath:filePath];
```

Comment:
What if this is a big data file? Do we want to offload this to a background queue?